package.json

```diff
@@ -82,10 +82,11 @@
     "pegjs": "^0.10.0",
     "preact": "^7.1.0",
     "preact-compat": "^3.9.4",
+    "prop-types": "^15.5.8",
```

Comment:
👍 

(This is what https://github.com/reactjs/prop-types#how-to-depend-on-this-package recommends for libs)